posthog/test/test_feature_flag.py

```diff
@@ -437,7 +437,6 @@ def test_setting_overrides_doesnt_balk_with_existing_overrides(self):
             self.assertEqual(len(res), 3)
             self.assertEqual(set([var[0] for var in res]), set([hash_key]))
 
-    @snapshot_postgres_queries(search_clauses=["SELECT", "INSERT"])
```

Comment:
Is this intended?

Reply:
Flaky snapshots, harming a few other PRs coupled with the CI not running on snapshot updates, so getting rid of both things in one 😅